resources/js/Admin/components/Form/CheckboxOptionsField.js

```diff
@@ -41,6 +41,9 @@ const CheckboxOptionsField = ({
       {options.length ? (
         <ul className="flex flex-wrap">
           {options.map((option, index) => {
+            // Remvoe "name" prefix from value when used for state data.
```

Comment:
Just a lil typo here!
```suggestion
            // Remove "name" prefix from value when used for state data.
```